widget_driver/example/lib/widgets/playground/playground_test_widget/playground_test_child_widget.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+import 'package:widget_driver/widget_driver.dart';
+
+import 'playground_test_child_widget_driver.dart';
+
+class PlaygroundTestChildWidget extends DrivableWidget<PlaygroundTestChildWidgetDriver> {
```

Comment:
The purpose of this `DrivableWidget` is to show case how you can use the new TestConfigProvider in your widgets tests to force a `DrivableWidget` to use the real driver.
So I created this `PlaygroundTestChildWidget` which has a driver which has no external dependencies.
This way, you can easily in the tests, configure the `PlaygroundTestChildWidget` to use the real driver instead of a testdriver. And the test will not crash since the real driver only has hardcoded strings. There is not mock-dependencies which you need to provided to it